crates/ty/build.rs

```diff
@@ -48,7 +49,8 @@ fn commit_info(workspace_root: &Path) {
         .arg("-1")
         .arg("--date=short")
         .arg("--abbrev=9")
-        .arg("--format=%H %h %cd %(describe)")
+        .arg("--format=%H %h %cd %(describe:tags)")
```

Comment:
Apparently a bare `describe` only works for annotated tags? This works for unannotated tags too.